clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -242,7 +253,7 @@
  * It isn't possible to mix both subscription to specific partitions (with no load balancing) and to topics (with load
  * balancing) using the same consumer instance.
  *
- * <h4>Managing Your Own Offsets</h4>
+ * <h4><a name="rebalancecallback">Managing Your Own Offsets</h4>
```

Comment:
Maybe "storing offsets outside Kafka"? Otherwise its not obvious how "Managing your own offsets" is different from "Manual offset control"...